### PR TITLE
feat: add links for daemon and jina openapi spec

### PR DESCRIPTION
**Pull request by rutujasurve94**
API reference Links to:

-  `api.jina.ai/daemon`
- `api.jina.ai/rest`

### CODE REVIEW

File: chapters/daemon/index.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Jina Daemon API Specification
+=============================
+
+[Link to Jina Daemon OpenAPI Specification](https://api.jina.ai/daemon/)
```
Comment (bwanglzu) [inline chapters/daemon/index.rst]: @rutujasurve94 this markdown style link won't work, please use rst link: `Link text <link URL>`_

**Review by bwanglzu — CHANGES_REQUESTED** [review]

**Review by bwanglzu — APPROVED** [review]
LGTM

File: chapters/daemon/index.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Jina Daemon API Specification
+=============================
+
+Link to Jina Daemon OpenAPI Specification <https://api.jina.ai/daemon/>
```
Comment (FionnD) [inline chapters/daemon/index.rst:4]: Simple and clean! I like it. 

One comment, can you convert this line (and line 4 in the other file) to active voice please? 

_Use this link to find the Jina Daemon OpenAPI Specification_

File: chapters/daemon/index.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+This is a REST interface for managing distributed Jina
```
Comment (FionnD) [inline chapters/daemon/index.rst:6]: Just add a full stop.

**Review by FionnD — CHANGES_REQUESTED** [review]
Clean and simple! Two small comments. There is no need for a second review, you can merge if the other reviews are happy.

**Pull request merged**